spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -3,6 +3,8 @@
 describe Idv::SessionsController do
   render_views
 
+  let(:provider) { 'https://rp1.serviceprovider.com/auth/saml/metadata' }
+  let(:authn_context) { Saml::Idp::Constants::LOA2_AUTHNCONTEXT_CLASSREF }
```

Comment:
Yes, thats a separate PR though.  I guess.  We should only support two possible values for LOA going forward right?